contact.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "780d6f7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "cb1759af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=['name','contact']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6d9b48a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "records=[['neel',98745612310],\n",
+    "         ['charvin',7894561237],\n",
+    "         ['shiv',8523697419],\n",
+    "         ['om',9123654789],\n",
+    "         ['meet',8596321478]\n",
+    "        ]\n",
+    "p=[]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "5081ea51",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ENTER THE NAME:dev\n",
+      "ENTER THE CONTACT:9362584172\n",
+      "ENTER THE NAME:bhavik\n",
+      "ENTER THE CONTACT:8524631972\n",
+      "ENTER THE NAME:ved\n",
+      "ENTER THE CONTACT:7985426319\n",
+      "ENTER THE NAME:raj\n",
+      "ENTER THE CONTACT:8596314259\n",
+      "ENTER THE NAME:jenil\n",
+      "ENTER THE CONTACT:9654871237\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(5):\n",
+    "    name=input(\"ENTER THE NAME:\")\n",
+    "    contact=int(input(\"ENTER THE CONTACT:\"))\n",
+    "    l=[name,contact]\n",
+    "    p.append(l)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "35b236a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('C:\\sqlite3\\contact.csv','w',newline='') as f:\n",
```

Comment:
without with can i open csv file?

Reply:
You can open the CSV file on Google Sheet, Notepad, or OpenOffice Calc.